apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -162,11 +163,16 @@ const SectionsTable = React.createClass({
               key={index}
             >
               <td style={{...styles.col, ...styles.sectionNameCol}}>
-                {isTeacher && (
+                {isTeacher && section.numberOfStudents > 0 &&(
                   <a href={section.linkToProgress} style={styles.link}>
                     {section.name}
                   </a>
                 )}
+                {isTeacher && section.numberOfStudents === 0 &&(
+                  <a href={`${codeOrgUrlPrefix}/teacher-dashboard#/sections/${section.id}/manage`} style={styles.link}>
+                    {section.name}
+                  </a>
+                )}
```

Comment:
It looks like the only thing that changes between these two elements is the href?

I think I'd prefer to either (a) precompute the href at the beginning of the render function and then just do:
```
{isTeacher &&
  <a href={sectionHref} style={styles.link}>{section.name}</a>
}
```
or (b) have a method for calculating the href
```
{isTeacher &&
  <a href={this.sectionHref()} style={styles.link}>{section.name}</a>
}
```